src/main.py

```diff
@@ -430,5 +452,9 @@ def main():
             traceback.print_exc()
             time.sleep(10)
 
+def concat_names(investors):
+    names = list(map(lambda i: "/u/" + i.name, investors))
```

Comment:
Just a little note, MIB is Python3.X oriented, so I would strongly suggest on changing python2 lambda expressions to python3 list comprehension

Reply:
I think it would be something like:

``` python
names = ["/u/" + x.name for x in investors]
```

Reply:
Will do. I'm a python noob so let me know if there is anything else I could do differently.

Reply:
I was just taking a quick look, everything looks fantastic! I will comments if I have some. Thank you for your work!